src/commands/generate/__tests__/generatorTest.ts

```diff
@@ -168,37 +173,29 @@ function testGenerateAllFilesAreTheSame(
 
         await generate(conjureDefinition, outputDir, typeGenerationFlags);
 
-        expectAllFilesAreTheSame(conjureDefinition, outputDir, actualTestCaseDir, typeGenerationFlags);
+        expectIdenticalDirectoryTrees(outputDir, actualTestCaseDir);
     };
 }
 
-function expectAllFilesAreTheSame(
-    definition: IConjureDefinition,
-    actualDir: string,
-    expectedDir: string,
-    typeGenerationFlags: ITypeGenerationFlags,
-) {
-    for (const type of definition.types) {
-        // We do not generate flavoured types for all aliases
-        if (type.type === "alias" && !isFlavorizable(type.alias.alias, typeGenerationFlags.flavorizedAliases)) {
-            continue;
-        }
-        const relativeFilePath = typeNameToFilePath(ITypeDefinition.visit(type, typeNameVisitor));
-        assertOutputAndExpectedAreEqual(actualDir, expectedDir, relativeFilePath);
+function expectIdenticalDirectoryTrees(actualDir: string, expectedDir: string) {
```

Comment:
The previous implementation was pretty crazy -- it would look at specific files rather than comparing the input/output directories as a whole. I fixed that.